src/WireSlaveRequest.cpp

```diff
@@ -68,7 +68,7 @@ bool WireSlaveRequest::request(uint8_t address)
 
     // copy payload bytes to rxBuffer
     rxIndex_ = 0;
-    while (unpacker.available()) {
+    while ((unpacker.available()) && (rxIndex_ < UNPACKER_BUFFER_LENGTH)) {
```

Comment:
Extra parentheses. Just write

`while (unpacker.available() && rxIndex_ < UNPACKER_BUFFER_LENGTH)`

Reply:
Yes of course. I would suggest that. 

```suggestion
    while (unpacker.available() && (rxIndex_ < UNPACKER_BUFFER_LENGTH)) {
```

Reply:
Here also you can commit my suggestion and after that merge it to your master, if you like ;)

Otherwise nice project.